### ISSUE TITLE
Fix for IE7/8 addEventListener

### ISSUE THREAD

**Issue by lyonlim** [dave1010/jquery-fast-click#17]
Replace

``` javascript
document.addEventListener('click', $.clickbuster.onClick, true);
```

with

``` javascript
    if (document.addEventListener){
      document.addEventListener('click', $.clickbuster.onClick, true); 
    } else if (document.attachEvent){
      document.attachEvent('onclick', $.clickbuster.onClick);
    }
```

**Comment by dave1010** [comment]
This looks good. Could you submit a pull request?

**Comment by dave1010** [comment]
Fixed in 8e748dce6a00ea4e4693bd35be4e4160f8f035a0
